homeassistant/components/simplefin/binary_sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Binary Sensor for SimpleFin."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from simplefin4py import Account
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SimpleFinConfigEntry
+from .entity import SimpleFinEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class SimpleFinBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Describes a sensor entity."""
+
+    value_fn: Callable[[Account], bool]
+
+
+SIMPLEFIN_BINARY_SENSORS: tuple[SimpleFinBinarySensorEntityDescription, ...] = (
+    SimpleFinBinarySensorEntityDescription(
+        key="possible_error",
+        translation_key="possible_error",
```

Comment:
Is this PR missing a `strings.json` entry?

Reply:
Good find! :)